### PR TITLE
add 'searchResolveModulesDirectories option

### PR DESCRIPTION
**Pull request by maxenglander**
Thanks for the helpful plugin @lpiepiora!

I added this option in my fork because I only want this plugin to handle my `bower_components`, and not for `node_modules`. Setting `searchResolveModulesDirectories` seemed like a simple way to achieve that. 

Perhaps there's a better way to do what I wanted or you won't deem this appropriate for merging, but sending a PR just in case there's not and you will.

### CODE REVIEW

**Comment by lpiepiora** [comment]
Hi @maxenglander,

thanks a lot for your contribution. In general it looks good to me, and I appreciate it has tests and such, so I could merge it to the main release, and release a new version. 

The only think I would prefer to change in your PR, is to have the flag set to `true` by default, because if I get it correctly, that would make the plugin behave the same way it behaves now. Unless you think this makes no sense, and the new behaviour should be the default one. What do you think?

File: lib/bower-plugin.js [hunk 1]
```diff
@@ -53,19 +54,20 @@ function BowerWebpackPlugin(options) {
   this.manifestFiles = opt.manifestFiles ? [].concat(opt.manifestFiles) : ["bower.json"];
   this.includes = opt.includes ? [].concat(opt.includes) : [/.*/];
   this.excludes = opt.excludes ? [].concat(opt.excludes) : [];
+  this.searchResolveModulesDirectories = opt.searchResolveModulesDirectories === false ? false : true;
```
Comment (maxenglander) [inline lib/bower-plugin.js]: The flag is false by default, contrary to what the JSDoc says. Sorry! Will clarify the JSDoc.

**Comment by maxenglander** [comment]
Hi @lpiepiora I definitely agree. The code actually sets it to `true` by default already, but I put the wrong thing in the JSDoc. Will fix that now.

**Comment by maxenglander** [comment]
Oh actually the JSDoc already sets it to `true` by default as well (I misread my own code) so I think we're all set here in terms of that.

File: lib/bower-plugin.js (hunk 1 above)
Comment (maxenglander) [inline lib/bower-plugin.js]: Er whoops it's actually `true` by default. Unless the user explicitly sets `false` in the `options`, is will search through `resolve.modulesDirectories`.

**Comment by lpiepiora** [comment]
Cool, just one last think, please rebase it into one commit and we should be good. Just in case you wonder how can this be done, this should help : https://help.github.com/articles/about-git-rebase/

**Comment by maxenglander** [comment]
Sure thing, done.

**Comment by lpiepiora** [comment]
Thanks, give me couple of minutes, and I'll release a new version to NPM

**Comment by lpiepiora** [comment]
Done, published as v0.1.8

**Pull request merged**